airtable/backup_airtable.py

```diff
@@ -2,44 +2,18 @@
 import datetime
 import os
 import tempfile
-import urllib
 import zipfile
 
 from boto.s3.connection import S3Connection
 from boto.s3.key import Key
-import requests
+import airtable
```

Comment:
Nice! Glad you pulled this stuff out of here.

I assume you verified it still works